src/common/NoResult/index.js

```diff
@@ -0,0 +1,11 @@
+import { StyledImg, Wrapper } from "./styled";
+import { StyledContainer } from "../Container/styled"
+import { StyledMainHeader } from "../MainHeader/styled";
+export const NoResult = ({query}) => (
+    <StyledContainer>
+        <StyledMainHeader title={`Sorry, there are no results for "${query}"`} />
+        <Wrapper>
+            <StyledImg />
+        </Wrapper>
+    </StyledContainer>
+);
```

Comment:
The code doesn't seem to be properly formatted. I highly recommend to set formatting on file save in VSC 😎

Reply:
Done :-)